documentation/hacker-guide/source/topics/making-a-release.rst

```diff
@@ -0,0 +1,50 @@
+******************
+Release Check-list
+******************
+
+There are a number of manual steps involved in making an Open Dylan release and
+it's easy to forget one or two. Hopefully we can automate some of these but for
+now here is a manual check-list.
+
+#. Let people know you plan to make a release. Use the Gitter channel for this.
+   Because the Dylan ncommunity is small there's no need to make a release
+   branch; people can just be careful what they commit until the release is
+   out.
+
+#. Test on supported platforms.
+
+   * Do a 3-stage boostrap.
+   * Run test suites and try to figure out if anything serious is broken.
+     (This is an art, currently, and we should make all our tests pass.)
+   * Do some ad-hoc testing.  (Anything specific to mention here?)
```

Comment:
Windows is different, I think only @housel knows how to do it.
Also do we need to check what version you have to have installed to bootstrap, e.g. will you be able to bootstrap 2019.2 if you have 2014.1 as your current Dylan?

Reply:
Yeah, I think I remember this exact problem happening in the past. i.e., the latest version wasn't usable for bootstrapping on a specific platform.  How does it look now?